ci_framework/roles/edpm_build_images/tasks/package.yml

```diff
@@ -0,0 +1,20 @@
+---
+- name: Package edpm-hardened-uefi image inside container image
+  args:
+    chdir: "{{ cifmw_edpm_image_builder_repo_path }}"
+  ansible.builtin.shell:
```

Comment:
Seems like buildah already logs to file :) ` --logfile` https://github.com/containers/buildah/blob/main/docs/buildah-build.1.md